CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-* Create V2 for `BakeIndex` to bake different types of indexes (minor)
+* Modify `BakeIndex` to bake multiple types of indexes (major)
```

Comment:
is this major? it seems backwards-compatible to me

Reply:
Hmm :thinking: There should be an error that `types` key is missing.
I tried to bake Calculus and I got this:

![image](https://user-images.githubusercontent.com/40228854/128151181-d6231a5c-a535-49f2-a5ad-9ef6517e9c70.png)

Reply:
I believe that error should not be happening now that a default value has been added to `BakeIndex#v1`

Reply:
You're right :wink: There is no error now. I changed log to `minor`.